src/shared/components/home/home.tsx

```diff
@@ -334,7 +335,7 @@ export class Home extends Component<any, HomeState> {
         {this.state.siteRes.site_view.local_site.site_setup && (
           <div className="row">
             <main role="main" className="col-12 col-md-8">
-              <div id="tagline" dangerouslySetInnerHTML={mdToHtml(this.state.tagline)}></div>
+              {this.state.tagline.isSome() && <div id="tagline" dangerouslySetInnerHTML={mdToHtml(this.state.tagline.unwrapOr(""))}></div>}
```

Comment:
Use `.match` here, you'll find a ton of examples in the code.

Reply:
okay fixed this one too.